plugins/SegmentEditor/templates/_segmentSelector.twig

```diff
@@ -56,10 +56,10 @@
                     </strong></div>
                 {{ 'General_And'|translate }} <div class="auto_archive"><strong>
                         <select class="auto_archive_select">
-                            {% if createRealTimeSegmentsIsEnabled %}
+                            {% if isBrowserArchivingEnabled %}
```

Comment:
This would need to be reverted to the original `if createRealTimeSegmentsIsEnabled `? If browser archiving is disabled, then you might still want to create a segment for "real time" if you use the segment for example only for the "visits log" or if you view it very rarely. It should just not be selected by default.